packages/app/pages/Api/agentRelationship.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * This exports a React element which displays all Agent Relationships
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-02-02
+ */
+
+import * as React from "react";
+import "./api.css"
+import getAllAgentRelationships from "../../../ui-bindings/AgentRelationship/getAllAgentRelationships";
+import getAgentRelationshipById from "../../../ui-bindings/AgentRelationship/getAgentRelationshipById";
+
+export const AgentRelationship = (props) => {
+  var agentRelationship = props.agentRelationship;
+  return(
+    <div>
+      <div>id: {agentRelationship.id}</div>
+      <div>subject id: {agentRelationship.subject.id}</div>
+      <div>object id: {agentRelationship.object.id}</div>
+      <div>relationship id: {agentRelationship.relationship.id}</div>
+      <br/>
+    </div>
+  );
+};
+
+const AgentRelationshipField = (props) => {
+  return(
+    <div>
+      <form onSubmit={props.onSubmitAction}>
+        Enter an Id: <input type="text" name="value" onChange={props.setAgentRelationship}/>
+        <input type="submit" value="query"/>
+      </form>
+    </div>
+  );
+};
+
+export const GetAllAgentRelationships = getAllAgentRelationships(({ agentRelationshipList, loading, error}) => {
+  return (
+    loading ? <strong>Loading...</strong> : (
+      error ? <p style={{color: "#F00"}}>API error</p> : (
+        <div>
+          {agentRelationshipList.map( (agentRelationship) => (<AgentRelationship key={agentRelationship.id} agentRelationship={agentRelationship}/>))}
```

Comment:
This line is really long. Consider breaking it into multiple lines

Reply:
Yup, good catch. I've made the change